va/va.h

```diff
@@ -795,6 +795,10 @@ typedef enum
      *  this setting also could be update by \c VAContextParameterUpdateBuffer
      */
     VAConfigAttribContextPriority       = 41,
+    /** \brief Add another pass to lookahead several frames before normal encode pass.
```

Comment:
if there are such capability reported, how to use it?

Reply:
Comments are added. If capability reports this feature can be supported, application can query capability for max frames to look ahead, then set look ahead depth no more than that max value to enable look ahead encoding.